### PR TITLE
signr-chrome now uses localStorage to store user identity

### PR DESCRIPTION
**Pull request by jlevesy**

### CODE REVIEW

**Comment by jlevesy** [comment]
@AlexisMontagne @phndiaye RTM ? If ok, merge and make sure to publish the new version ;)

File: src/module/identity.coffee [hunk 1]
```diff
@@ -0,0 +1,37 @@
+$ = require('jquery')
+
+IdentityProvider = (page, menus, storage) ->
+  @page = page
+  @menus = menus
+  @storage = storage
+  return
+
+IdentityProvider.prototype =
```
Comment (AlexisMontagne) [inline src/module/identity.coffee]: `class` we are using CoffeeScript

File: src/module/identity.coffee [hunk 2]
```diff
@@ -0,0 +1,37 @@
+$ = require('jquery')
+
+IdentityProvider = (page, menus, storage) ->
```
Comment (AlexisMontagne) [inline src/module/identity.coffee]: this is an `initializer`

File: src/module/injector.coffee [hunk 3]
```diff
@@ -0,0 +1,17 @@
+SignatureInjector = (page, menus) ->
```
Comment (AlexisMontagne) [inline src/module/injector.coffee]: idem

**Comment by jlevesy** [comment]
@AlexisMontagne RTM ?

**Comment by AlexisMontagne** [comment]
LGTM!

**Pull request merged**